articles/100_ament.md

```diff
@@ -20,34 +20,93 @@ author: '[Dirk Thomas](https://github.com/dirk-thomas)'
 Original Author: {{ page.author }}
 
 
+## Preface
+
+ROS is developed with a federated model.
+While this comes with a lot of advantages it makes building a set of interdependent packages more complex.
+
+Commonly a user has to build interdependent packages in the correct topological order.
+And for each package the steps include reading the documentation of a package, then building and installing it following a few build system specific commands and then setting up the environment for other packages to use it.
```

Comment:
My fifth grade teacher would say that starting a sentence with `And` is not a good idea. Maybe this structure would be more formal:

```
In order to build a set of interdependent packages the user must build them in the correct topological order.
The user must look at the documentation of each package to determine the dependencies, being careful to build those packages before hand, and to determine the instructions for building that package.
Finally the user must build and install each package and do any environment configuration before proceeding to build the next package.
```